src/restLevelClient/generateRestLevel.ts

```diff
@@ -77,6 +79,8 @@ export async function generateRestLevelClient() {
   // Save the source files to the virtual filesystem
   project.saveSync();
   const fs = project.getFileSystem();
+  const pathToClear = outputPath ? path.join(outputPath, srcPath) : srcPath;
```

Comment:
I think Autorest can do this if you pass `--clear-output-folder=true`, maybe we don't need to do this in here. Can you try removing these changes and in the generate-swaggers script add   `--clear-output-folder=true`?

Reply:
Talked about this offline. We do this same thing in HLC generator, we should be fine

Reply:
We had a similar discussion for HLC about this before https://github.com/Azure/autorest.typescript/issues/1166#issuecomment-905033088